### ISSUE TITLE
bug: when absolute file path is given in Flags, doing a replace duplicates file contents

### ISSUE THREAD

**Issue by MagicDuck** [MagicDuck/grug-far.nvim#124]

**Comment by MagicDuck** [comment]
fixed in https://github.com/MagicDuck/grug-far.nvim/pull/127

**Comment by MagicDuck** [comment]
@adinhodovic no, I cannot reproduce it. I am on ripgrep 14.1.0. You will need to provide more accurate reproduction steps for me to take a look...

**Comment by adinhodovic** [comment]
@MagicDuck my config is fairly simple:
```
	{
		"MagicDuck/grug-far.nvim",
		opts = {
			transient = true,
			engines = {
				ripgrep = {
					path = "/usr/bin/rg",
				},
			},
		},
		keys = {
			{
				"<leader>sv",
				function()
					require("grug-far").with_visual_selection({
						prefills = {
							flags = vim.fn.expand("%"),
						},
					})
				end,
				mode = { "v" },
				desc = "Grug: Replace word in current file visually selected",
			},
		},
	},
```

I'm visually selecting and using sv.

As you see in the image the full result is here:

![Image](https://github.com/user-attachments/assets/149f6f31-5240-4d64-aa4c-1fde66e9ab24)

It's duplicating full file content. My expected behaviour is as below:

If I remove the flags it works:

![Image](https://github.com/user-attachments/assets/a8162a8d-b785-470b-944d-926282056121)

Rg 14.1.1, latest grug (2 days ago 1.5.6)

**Comment by MagicDuck** [comment]
oh, I see what's going on, the file path is going into `Flags` instead of `Paths`. Should be an easy fix

**Comment by MagicDuck** [comment]
actually, there is nothing for me to fix you just need to change your config:
```
flags = vim.fn.expand("%"),
```
to
```
paths = vim.fn.expand("%"),
```
I am guessing you had your config from older times, back when we didn't have `Paths` 😄

**Comment by MagicDuck** [comment]
no problem at all 😄
